src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/utils.ts

```diff
@@ -107,3 +116,10 @@ export const sanitizeSearchTime = (searchTime: SearchTime) =>
   searchTime.option === 'now'
     ? {...searchTime, date: new Date().toISOString()}
     : searchTime;
+
+export const areDefaultFiltersSelected = (
+  transportModes?: TransportModeFilterOptionWithSelectionType[],
+): boolean => {
+  if (!transportModes) return false;
+  return !transportModes.some((tm) => tm.selectedAsDefault !== tm.selected);
```

Comment:
I think `transportModes.every((tm) => tm.selectedAsDefault === tm.selected)` is quite a bit easier to read than using `some` which needs two negations in the statement. And I think they should end up the same 😅 At least we have some tests to check it.